test/unit/Command.spec.ts

```diff
@@ -104,17 +104,6 @@ describe('Command.listorders.formatOrders', () => {
     const output = formatOrders(jsonOrders);
     expect(output.length).to.equal(1);
     expect(output[0].pairId).to.equal('LTC/BTC');
-    const expectedFirstRow = [
```

Comment:
This test doesn't do very much after these lines are removed. I'm thinking it might be nice to have some tests to verify the sorting and the limit parameter in the simulation tests, but that can be a separate PR.